Sources/PointFree/TODO.swift

```diff
@@ -463,3 +477,73 @@ public func require2<A, B, Z>(_ x: T3<A, B?, Z>) -> T3<A, B, Z>? {
 public func lower<A>(_ tuple: Tuple1<A>) -> A {
   return get1(tuple)
 }
+
+import Cryptor
+
+extension PartialIso where A == String, B == String {
+  public static func decrypted(withSecret secret: String) -> PartialIso<String, String> {
```

Comment:
this is the general purpose `decrypted` partial iso in which you provide a secret. this could be moved to a support lib in swift-web